### ISSUE TITLE
3 Таб Задать вопрос меняем на "Условия доставки", здесь предусмотреть область под текст

### ISSUE THREAD

**Issue by Raken85** [Raken85/allacouture.com#13]

**Comment by uralclick** [comment]
Подскажи как будет редактироваться текст этот?

**Comment by Raken85** [comment]
можно в режиме правки на детальной странице http://prntscr.com/og97hm или сам файл на странице http://allacouture/bitrix/admin/fileman_file_edit.php?path=%2Finclude%2Fterms_of_delivery.php&full_src=Y&site=s2&lang=ru&&filter=Y&set_filter=Y

**Comment by uralclick** [comment]
т.е если на детальной странице правишь, то у конкретного товара меняется? если в файле, то для всех ?

**Comment by Raken85** [comment]
Нет, в данный момент правится один файл для всех страниц, просто показано два способа как его править. Если надо уникальные условия для товаров, то это надо доделывать.

**Comment by uralclick** [comment]
Не могу протестить, в режиме правки не вижу пункта "редактировать условия доставки HTML" https://yadi.sk/i/FJhnA-22ZRRPuQ

ссылку тоже не получается открыть

**Comment by Raken85** [comment]
переименовал пункт в режиме правки https://i.imgur.com/sNUkUIF.png

**Comment by uralclick** [comment]
блин, почему у меня нет этого пункта в режиме правки ?))
https://yadi.sk/i/67egXEdWV_3vOA

**Comment by Raken85** [comment]
при сброшенном кеше появилось https://i.imgur.com/v5rRam7.png
